src/pages/pipeline_health.astro

```diff
@@ -1,21 +1,296 @@
 ---
 import PageLayout from '@layouts/PageLayout.astro';
+import { remote_workflows as pipelines } from '@public/pipelines.json';
 ---
 
 <PageLayout
     title="Pipeline health"
     subtitle="Check GitHub settings for all nf-core repositories"
     mainpage_container={false}
 >
-    <div class="iframe-container w-100 d-flex align-items-start vh-100">
-        <i class="mt-5 m-auto text-success fa-regular fa-spinner-third fa-spin fa-3x"></i>
-        <iframe
-            class="d-none"
-            src="https://oldsite.nf-co.re/pipeline_health"
-            onload="this.classList.remove('d-none'); this.previousElementSibling.classList.add('d-none');"
-            title="pipeline_heath"
-            style="border:none;"
-            width="100%"
-            height="100%"></iframe>
+    <div class="container-fluid main-content w-100 mw-100">
+        <div class="table-responsive">
+            <h2>Pipelines</h2>
+            <table class="table table-hover table-sm">
+                <thead>
+                    <tr>
+                        <th class="small fw-normal text-nowrap">Pipeline Name</th>
+                        <th class="small fw-normal text-nowrap" title="Has at least one release" data-bs-toggle="tooltip" data-bs-placement="top">Released</th>
```

Comment:
Can probably remove all these verbose `data-bs-tooltip` type statements with a small Javascript snippet at the bottom of the file if we want to clean up the HTML a bit.

Not sure if the `small fw-normal text-nowrap` will also work on a parent element?

Reply:
I haven't payed too much attention to the headers yet, tbh I literally just copy-pasted the elements from https://oldsite.nf-co.re/pipeline_health 😄

Reply:
Hah, in that case I literally only have myself to blame for this 😆